server/app/Module.scala

```diff
@@ -0,0 +1,9 @@
+import com.google.inject.AbstractModule
+import com.google.inject.name.Names.named
+
+import scala.util.Random.nextInt
+
+class Module extends AbstractModule {
+  override def configure() =
```

Comment:
Agree with you, but in such micro applications it's not a problem, can revert `AbstractModule.configure()` easily